Directory.Build.props

```diff
@@ -16,7 +16,7 @@
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
 
     <!-- Package info -->
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
```

Comment:
With this change do we still need the `LICENSE` file?

Reply:
- We need it in the git repository
- I think we need it in the package as we forked from an MIT project, we need to copy the license file, but I'm not a licensing expert...

    > The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.